README.md

```diff
@@ -53,6 +53,17 @@ and health-checking mechanisms.
   ```
 
 * Install puppet-agent (&ge; 6.13.0)
+  
+  Check facter version details, if already installed / exist in system
+  ```sh
+  facter -v
+  ls -ll /usr/bin/facter
```

Comment:
I don't think it makes sense to run this `ls`.

Reply:
yes, removed it as it gets covered in below command.